src/preloadjs/LoadQueue.js

```diff
@@ -405,6 +405,15 @@ this.createjs = this.createjs || {};
 		this._crossOrigin = crossOrigin;
 
 		/**
+		 * An array of the plugins registered using {{#crossLink "LoadQueue/installPlugin"}}{{/crossLink}}.
+		 * @property _plugins
+		 * @type {Array}
+		 * @private
+		 * @since 0.6.1
+		 */
+		this._plugins = [];	// TODO this might make a good getter / setter
```

Comment:
Remove TODO .. since we already have installPlugin, no need for a getter/setter here.